templates/ingress.yaml

```diff
@@ -37,6 +37,7 @@ metadata:
     app.kubernetes.io/part-of: ingress-nginx
 data:
   22: "{{ .Release.Namespace }}/envproxy:2222"
+  5349: "{{ .Release.Namespace }}/cemanager:5349"
```

Comment:
Did we find out we _don't_ need any proxy protocol here?

Reply:
We don't!

It's kinda a separate issue requiring the PROXY proto TBH... it allows us to fully forward external IPs which we should have for auditing, but isn't a neeeeed right now.